### PR TITLE
Migrate code from googleapis/python-datalabeling

### PR DESCRIPTION
**Pull request by ivanmkc**
- chore: move samples from python-docs-sample to here (#29)
- feat!: migrate to use microgen (#34)
- samples: updated requierement.txt and enabled list sample test (#48)
- chore: added helpful instruction comment on lable_image sample (#52)
- feat: add common resource helper paths; expose client transport (#49)
- chore: currently datalabeling is limited to due to covid 19 (#61)
- chore: update templates (#63)
- chore: Updating resources and configuration for presubmits and test environments (#72)
- chore: disabling datalabeling tests due to service limited exception (#81)
- chore: migrate to owl bot (#96)
- chore(deps): update dependency pytest to v6.2.4 (#97)
- chore: new owl bot post processor docker image (#103)
- chore(deps): update dependency google-cloud-datalabeling to v1.1.0 (#106)
- chore(deps): update dependency backoff to v1.11.0 (#118)
- chore(deps): update dependency google-cloud-datalabeling to v1.2.0 (#119)
- chore(deps): update dependency backoff to v1.11.1 (#120)
- feat: add Samples section to CONTRIBUTING.rst (#124)
- chore(deps): update dependency google-cloud-datalabeling to v1.2.1 (#126)
- chore(deps): update dependency google-cloud-datalabeling to v1.2.2 (#132)
- chore: fix INSTALL_LIBRARY_FROM_SOURCE in noxfile.py (#134)
- chore: drop mention of Python 2.7 from templates (#136)
- chore(deps): update dependency pytest to v6.2.5 (#141)
- chore: blacken samples noxfile template (#146)
- chore: fail samples nox session if python version is missing (#153)
- chore(python): Add kokoro configs for python 3.10 samples testing (#158)
- chore(deps): update dependency google-cloud-datalabeling to v1.3.0 (#162)
- chore(deps): update dependency google-cloud-datalabeling to v1.4.0 (#168)
- chore(samples): Add check for tests in directory (#181)
- chore(python): Noxfile recognizes that tests can live in a folder (#185)
- chore(deps): update dependency pytest to v7 (#193)
- chore(deps): update dependency pytest to v7.0.1 (#195)
- chore(deps): update dependency google-cloud-datalabeling to v1.5.0 (#196)
- chore: use gapic-generator-python 0.63.2 (#194)
- chore: use gapic-generator-python 0.63.4 (#197)
- chore: update copyright year to 2022 (#198)
- chore: Adding support for pytest-xdist and pytest-parallel (#207)
- chore(deps): update dependency google-cloud-datalabeling to v1.5.1 (#210)
- chore(deps): update dependency pytest to v7.1.0 (#212)
- chore(deps): update dependency pytest to v7.1.1 (#213)
- chore(python): use black==22.3.0 (#218)
- chore: use gapic-generator-python 0.65.1 (#228)
- chore(python): add nox session to sort python imports (#229)
- chore(deps): update dependency pytest to v7.1.2 (#232)
- chore(deps): update dependency backoff to v2 (#233)
- chore(deps): update dependency backoff to v2.0.1 (#235)
- fix: require python 3.7+ (#251)
- chore(deps): update all dependencies (#245)
- chore(deps): update all dependencies (#257)
- chore(deps): update dependency google-cloud-datalabeling to v1.6.1 (#263)
- chore(deps): update dependency pytest to v7.1.3 (#272)
- chore: Bump gapic-generator-python version to 1.3.0 (#273)
- chore: detect samples tests in nested directories (#276)
- chore(deps): update dependency google-cloud-datalabeling to v1.6.2 (#280)
- chore(deps): update dependency backoff to v2.2.1 (#281)
- chore(deps): update dependency google-cloud-datalabeling to v1.6.3 (#284)
- chore(deps): update dependency pytest to v7.2.0 (#285)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by ivanmkc** [comment]
CLA issues caused by previous committers: https://opensource.corp.google.com/github/prinfo?q=%2Fgithub.com%2FGoogleCloudPlatform%2Fpython-docs-samples%2Fpull%2F8456

Should I squash?

File: datalabeling/snippets/noxfile.py [hunk 1]
```diff
@@ -0,0 +1,312 @@
+# Copyright 2022 Google LLC
```
Comment (dandhlee) [inline datalabeling/snippets/noxfile.py:1]: This file is not needed. Please remove.

File: appengine/standard/ndb/cache/README.md [hunk 2]
```diff
@@ -1,16 +0,0 @@
-## App Engine Datastore NDB Cache Samples
```
Comment (dandhlee) [inline appengine/standard/ndb/cache/README.md:1]: Are these files purposely removed?

File: datalabeling/AUTHORING_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1 @@
+See https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md
```
Comment (dandhlee) [inline datalabeling/AUTHORING_GUIDE.md:1]:
```suggestion
See https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md
```

File: datalabeling/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1 @@
+See https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/CONTRIBUTING.md
```
Comment (dandhlee) [inline datalabeling/CONTRIBUTING.md:1]:
```suggestion
See https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/CONTRIBUTING.md
```

**Review by dandhlee — CHANGES_REQUESTED** [review]

**Comment by nicain** [comment]
@ivanmkc I don't think generated samples are in-scope for the migration. Ill ping the chat line and confirm.

File: appengine/standard/ndb/cache/README.md (hunk 2 above)
Comment (ivanmkc) [reply appengine/standard/ndb/cache/README.md:1]: I don't know why or how they where removed..

Comment (ivanmkc) [reply appengine/standard/ndb/cache/README.md]: Fixed.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Could you add a CODEOWNERS and blunderbuss for these files? Otherwise LGTM.

**Comment by ivanmkc** [comment]
@dandhlee can you re-review?

File: .github/blunderbuss.yml [hunk 5]
```diff
@@ -108,7 +108,11 @@ assign_issues_by:
   - 'api: translate'
   to:
   - nicain
-
+- labels:
+  - 'api: datalabeling'
+  to:
+  - python-sample-reviewers
```
Comment (dandhlee) [inline .github/blunderbuss.yml:114]:
```suggestion
  - GoogleCloudPlatform/python-samples-reviewers
```

**Comment by ivanmkc** [comment]
@dandhlee needs an override.

**Review by dandhlee — APPROVED** [review]
LGTM. Overriding CLA and admin merging.

**Pull request merged**